tester_app/client/assets/stylesheets/_bootstrap-variables-customization.scss

```diff
@@ -0,0 +1,18 @@
+// This file is used as part of the bootstrap-sass-loader customization
```

Comment:
@justin808 but it's not rails-specific either, this gets loaded even in HMR server